index.css

```diff
@@ -504,3 +563,346 @@ textarea {
 .formato {
   height: 400px;
 }
+
+@media (min-width: 768px) {
+  nav {
+    display: none;
+  }
+
+  .active {
+    color: #fff;
+    width: 92px;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    text-decoration: none;
+    margin: 28px 0 0 120px;
+  }
+
+  .topnav {
+    display: flex;
+    justify-content: space-between;
+    color: #fff;
+    background: #3e3c3c url(Desktop/mail.png) 91% 62% no-repeat;
+    height: 92px;
+  }
+
+  .topnav #myLinks {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #f2f2f2;
+    padding: 14px 50px 0 0;
+    margin-right: 110px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .roger {
+    height: 870px;
+    display: flex;
+    padding: 0 0 0 37px;
+    background: #1c1a19 url(Desktop/muñeco2.png) 100% 125% no-repeat;
+  }
+
+  .pad {
+    width: 566px;
+    height: 160px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    margin: 110px 80px;
+    color: #f55800;
+  }
```

Comment:
- [ ] Kindly make sure your fonts of the headline is close to that of the Figma template as much as possible

| Your design | Figma template |
| ------------- | ------------- |
| ![Capture](https://user-images.githubusercontent.com/85095702/160021232-b9356991-af32-426e-8f43-8f2764b5e433.PNG) | ![figma](https://user-images.githubusercontent.com/85095702/160021321-a2b5330b-a4fa-4faa-86f7-322271f61628.PNG) |

- [ ] Also kindly fix the margins of the social links to match that of the template